source/index.html

```diff
@@ -1,30 +1,258 @@
+        <div>
+          <img class="display465" src="./assets/derouins_small.png">
+          <img src="./assets/derouins.png">
+          <h5>Derouins Auto Body</h5>
+          <p>Derouin’s Auto Shop was my first paid job. The client was simply looking for a fresh look and to make the site mobile friendly. I created the design and built the site on Wordpress, allowing the client to make updates and perform common maintenance tasks.</p>
+          <a class="black-button" href="http://www.derouins.com/">website</a>
+        </div>
+      </div>
+    </section>
+    <section id="contact" class="email-me">
+      <div>
+        <h4>You have work a do?</h4>
+        <p>Something my son always asks when I head out the door in the morning. Well if you do, drop me a line. Let's chat about your upcoming project. See if I can help. I’ll bet you my grandma's secret pot pie recipe that I can. Spoiler alert, it's delicious.</p>
+        <a class="black-button " href="mailto:mark.m.hayes1@gmail.com">contact</a>
+      </div>
+    </section>
+    <section class="find-us">
+      <div class="line-break">
+        <hr class="big-line">
+        <h3 class="title">Yooper Made</h3>
+        <hr class="small-line ">
+      </div>
+      <div class="find-us-content">
+        <div>
+          <p> We know a thing or two about long distance relationships.</p>
+          <div class="location">
+            <address>Pot Pie Digital, LLC
+              <br>223 Genesee St.
+              <br>Marquette, Mi 49855.</address>
+            <a href="tel:906-322-3097 ">906-322-3097</a>
+            <a href="mailto:mark.m.hayes1@gmail.com "> or EMAIL</a>
+          </div>
+        </div>
+        <iframe src="https://www.google.com/maps/embed?pb=!1m14!1m12!1m3!1d1139615.7861125765!2d-86.73222554127625!3d46.620893562865206!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!5e0!3m2!1sen!2sus!4v1526918681378 " width="600 " height="250 " frameborder="0 " style="border:0 " allowfullscreen></iframe>
+      </div>
+    </section>
+    <footer>
+      <img src="./assets/pie.png">
+      <div class="foot-nav">
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Services</h4></li>
+            <li>Web Design</li>
+            <li>Development</li>
+            <li>Graphic Design</li>
+            <li>Branding</li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Company</h4></li>
+            <li>
+              <a href="#about">About</a>
+            </li>
+            <li>
+              <a href="resume.html">Resume</a>
+            </li>
+            <li>
+              <a href="#projects">Projects</a>
+            </li>
+            <li>
+              <a href="#contact">Contact</a>
+            </li>
+          </ul>
+        </div>
+        <div class="footer-menu">
+          <ul>
+            <li><h4>Links</h4></li>
+            <li>
+              <a href="https://www.facebook.com/" target="_blank">Facebook</a>
```

Comment:
Probably want these to link to something, not just like Facebook (or Instagram or Linkedin [with two little `i`s]) in general.